test/YieldCrabLever.t.sol

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
Would need to hear from @alcueca if he thinks these test files need to conform to the new spec first.